src/Elastic.Apm/Elastic.Apm.csproj

```diff
@@ -20,4 +20,9 @@
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.9.0" />
     <PackageReference Include="Ben.Demystifier" Version="0.1.6" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Configuration">
+      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.6.1\System.Configuration.dll</HintPath>
```

Comment:
Reference without `HintPath` should also works fine. `<Reference Include="System.Configuration" />`

Reply:
done